spacecmd/src/bin/spacecmd

```diff
@@ -156,7 +156,8 @@ if __name__ == '__main__':
         if shell.options.server:
             shell.config['server'] = shell.options.server
         else:
-            shell.config['server'] = get_localhost_fqdn()
+            shell.config['server'] = "localhost"
+            shell.config['nossl'] = True
```

Comment:
Spacecmd is also part of the client tools, and this version will also connect against 4.3. Connect from external should not be a problem. On SUMA 4.3 we do not update to this version. So i think should not affect 4.3.